src/frontend/src/binder/relation/table_or_source.rs

```diff
@@ -70,14 +70,32 @@ impl BoundSource {
 }
 
 impl Binder {
+    pub fn bind_catalog_relation_by_object_name(
+        &mut self,
+        object_name: ObjectName,
+        bind_creating_relations: bool,
+    ) -> Result<Relation> {
+        let (schema_name, table_name) =
+            Binder::resolve_schema_qualified_name(&self.db_name, object_name)?;
+        self.bind_catalog_relation_by_name(
+            None,
```

Comment:
Before this PR, we called `bind_relation_by_name` to bind the relation. We set the parameter of `allow_cross_db` to false in that function call:
```
        if let Ok(relation) = binder.bind_relation_by_name(object_name.clone(), None, None, false) {
```

If we trace the codepath, this will let the db name resolve to the binder's internal context db_name. So we can just pass in `None` here.

Reply:
Fixed https://github.com/risingwavelabs/risingwave/pull/22205/commits/3881c3f09ebd7c617f35e6081a2aec1e859b98cf